Gemfile.lock

```diff
@@ -185,29 +185,10 @@ GEM
       railties (>= 5.0.0)
     faker (2.21.0)
       i18n (>= 1.8.11, < 2)
-    faraday (1.10.0)
-      faraday-em_http (~> 1.0)
-      faraday-em_synchrony (~> 1.0)
-      faraday-excon (~> 1.1)
-      faraday-httpclient (~> 1.0)
-      faraday-multipart (~> 1.0)
-      faraday-net_http (~> 1.0)
-      faraday-net_http_persistent (~> 1.0)
-      faraday-patron (~> 1.0)
-      faraday-rack (~> 1.0)
-      faraday-retry (~> 1.0)
+    faraday (2.3.0)
```

Comment:
While the version bump here is significant it looks to be reasonable and is based on update of dependencies for `octokit`.

Reply:
Looks like this update now outputs to msfconsole on bootup by default which is annoying 😄 
```
$ bundle exec ruby ./msfconsole -q
To use retry middleware with Faraday v2.0+, install `faraday-retry` gem <-------
[*] Using configured payload windows/meterpreter/reverse_tcp
msf6 exploit(windows/smb/kerberos_test) > 
```

Tracked it down to octokit:
https://github.com/octokit/octokit.rb/blame/dd622a38073f383e7e355f6897ffed60b6198926/lib/octokit/default.rb#L7-L15

We can silence this output with OCTOKIT_SILENT:
https://github.com/octokit/octokit.rb/blob/94925391160987c74bd0e4dcc221255e1f0ba842/lib/octokit/warnable.rb#L8-L16

But that seems like a workaround, I'm tempted to say Octokit is either logging out for behavior we don't explicitly rely on - or if there's now a subtle bug for the code that relies on Octokit's on retry functionality. Would have to poke further. I believe it's only the pull request finder that uses this functionality.

Reply:
Yeah, looks like we'd be dropping the default retry behavior that used to be present - and now it explicitly requires a separate gem dependency to work:

https://github.com/octokit/octokit.rb/blame/ff8dc9c181e84de008c973db1745a65d895c1f9c/lib/octokit/default.rb#L33-L37

Reply:
PR to add the gem: https://github.com/rapid7/metasploit-framework/pull/16662